taxi/views.py

```diff
@@ -1,5 +1,6 @@
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render
+from django.urls import reverse_lazy
 from django.views import generic
 from django.contrib.auth.mixins import LoginRequiredMixin
```

Comment:
Code Style: Please group your imports using `()` instead of using a backslash. For example:

python
from django.contrib.auth.mixins import (
    LoginRequiredMixin,
)
```